dist/taginfo.json

```diff
@@ -312,10 +345,14 @@
     {"key": "short_name:ja"},
     {"key": "social_centre:for", "value": "veterans"},
     {"key": "sport", "value": "golf"},
+    {"key": "sport", "value": "yoga"},
+    {"key": "supervised", "value": "no"},
     {"key": "takeaway", "value": "only"},
     {"key": "takeaway", "value": "yes"},
     {"key": "tourism", "value": "hotel"},
     {"key": "tourism", "value": "motel"},
+    {"key": "trade", "value": "building_supplies"},
```

Comment:
another weird one 👀

Reply:
https://github.com/osmlab/name-suggestion-index/commit/0288e553c7debac15de8de616577bad6c4887ea7#diff-87db3b2155a8ad976aeda84255eabc22

Reply:
I guess this one is alright.  I didn't realize we had `shop/trade` now.